mne/preprocessing/ica.py

```diff
@@ -2084,3 +2101,230 @@ def _band_pass_filter(ica, sources, target, l_freq, h_freq, verbose=None):
         raise ValueError('Must specify both pass bands')
 
     return sources, target
+
+
+@verbose
+def corrmap(icas, template, threshold="auto", label="bads",
+            plot=True, copy=False, ch_type="eeg"):
```

Comment:
How about we always return the figure and drop returning the IC objects (so it always modifies in place, if it modifies)? Seems simpler to me. Heterogenous PLUS variable returns sounds confusing to me.

In the original algorithm, it modifies iff a label is set.